### PR TITLE
pod: exit with make's exitcode

### PR DESCRIPTION
**Pull request by imjasonh**

### CODE REVIEW

File: pkg/commands/pod.go [hunk 1]
```diff
@@ -56,6 +56,10 @@ func cmdPod() *cobra.Command {
 
 			arch := types.ParseArchitecture(arch).ToAPK()
 
+			if arch != "armv7" {
+				log.Fatalf("arch was %s, want arm/v7", arch)
```
Comment (kaniini) [inline pkg/commands/pod.go:60]: this seems unrelated?

Comment (imjasonh) [reply pkg/commands/pod.go:60]: Whhhooooops

File: pkg/commands/pod.go [hunk 2]
```diff
@@ -143,8 +143,8 @@ func cmdPod() *cobra.Command {
 set -euo pipefail
 # Download all packages so we can avoid rebuilding them.
 mkdir -p ./packages/
-gsutil -m rsync -r %s ./packages/
-`, srcBucket)},
+gsutil -m rsync -r %s%s ./packages/ || true
```
Comment (imjasonh) [inline pkg/commands/pod.go:146]: This also makes us only fetch packages for the arch we're going to build (and don't fail if that doesn't work)

**Review by kaniini — APPROVED** [review]

**Review by kaniini — APPROVED** [review]
